app/connection/NetworkMonitor.ts

```diff
@@ -40,9 +40,6 @@ class NetworkMonitor {
                 this.establishConnection();
             });
             window.addEventListener('offline', () => this.setNetworkStatus('offline'));
-        } else {
-            // default to always online and fallback to WS checks
-            navigator.onLine = true;
```

Comment:
@ahmadtaimoor-deriv please elaborate why don't we need this anymore? :)

Reply:
`navigator.onLine = true;`

navigator.onLine is readonly and cannot be explicitly set. It is controlled by the browser. Setting onLine to true will not have any effect . 
https://developer.mozilla.org/en-US/docs/Web/API/Navigator/onLine

Reply:
https://www.w3schools.com/jsref/prop_nav_online.asp (it says it is readonly)
@maryia-deriv , I think it needs to be removed from the deriv-app as well. Setting a readonly property to anything will not have any affect . 
Can also try changing it's value in a code snippet
https://www.w3schools.com/jsref/tryit.asp?filename=tryjsref_nav_all
I confirmed it with quality team as well. I'll create a PR to remove it from deriv-app as well.